eng/pipelines/aggregate-reports.yml

```diff
@@ -18,29 +27,20 @@ jobs:
       vmImage: 'windows-2019'
 
     steps:
-      - task: UseDotNet@2
-        displayName: 'Use .NET Core sdk $(DotNetCoreSDKVersion)'
-        inputs:
-          version: '$(DotNetCoreSDKVersion)'
+      - download: net-core
+        artifact: packages
+        patterns: '*'
 
       - template: /eng/common/pipelines/templates/steps/verify-links.yml
         parameters:
           Directory: ""
 
-      - pwsh: mkdir '$(System.ArtifactsDirectory)/BuildArtifacts'
+      - pwsh: |
+          mkdir "$(System.ArtifactsDirectory)/BuildArtifacts"
+          ls "$(PIPELINE.WORKSPACE)/net-core/packages"
+          Copy-Item -Path "$(PIPELINE.WORKSPACE)/net-core/packages/*" -Destination "$(System.ArtifactsDirectory)/BuildArtifacts"
         displayName: Create Artifact Directory
 
-      - task: DownloadPipelineArtifact@2
-        displayName: 'Download Pipeline Artifact'
-        inputs:
-          buildType: specific
-          project: $(TargetPipelineProjectId)
```

Comment:
We might be able to delete these variables in the pipeline once your change goes in.